### PR TITLE
refactor: front page updates

### PR DESCRIPTION
**Pull request by ydcjeff**

### CODE REVIEW

**Comment by vfdev-5** [comment]
@ydcjeff let's also put Quansight Labs logo as here: https://labs.quansight.org/blog/2020/09/pytorch-ignite/#Project-news

**Comment by vfdev-5** [comment]
Looks nice! Thanks, Jeff. I forgot to mention, can we still have a section as it was on the front page ?

**Comment by ydcjeff** [comment]
> Looks nice! Thanks, Jeff. I forgot to mention, can we still have a section as it was on the front page ?

Which section?

**Comment by vfdev-5** [comment]
Ecosystem, I mean

**Comment by vfdev-5** [comment]
Thanks @ydcjeff , can we also add this 
```
If your project implements a paper, represents other use-cases not covered in our official tutorials, Kaggle competition's code, or just your code presents interesting results and uses Ignite. We would like to add your project to this list, so please open an issue here: https://github.com/pytorch-ignite/pytorch-ignite.ai/issues with brief description of the project.
```
to the ecosystem page

File: content/ecosystem.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+title: "Ecosystem"
+description: Ecosystem around PyTorch-Ignite
+---
+
+> If your project implements a paper, represents other use-cases not covered in our official tutorials, Kaggle competition's code, or just your code presents interesting results and uses PyTorch-Ignite. We would like to add your project to this list, so please open an issue here: https://github.com/pytorch-ignite/pytorch-ignite.ai/issues with brief description of the project.
```
Comment (vfdev-5) [inline content/ecosystem.md:6]:
```suggestion
If your project implements a paper, represents other use-cases not covered in our official tutorials, Kaggle competition's code, or just your code presents interesting results and uses PyTorch-Ignite. We would like to add your project to this list, so please open an issue here: https://github.com/pytorch-ignite/pytorch-ignite.ai/issues with brief description of the project.
```

File: content/ecosystem.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+## Research Papers
```
Comment (vfdev-5) [inline content/ecosystem.md:8]:
```suggestion
The list below is unexhausted list of references that we are aware of.

## Research Papers
```

**Review by vfdev-5 — APPROVED** [review]
LGTM, thanks Jeff!

**Comment by vfdev-5** [comment]
@ydcjeff you can merge this one. Can you please create another PR to see how it is rendered if we put "Simple Engine and Event System", "Rich Handlers" in the same boxes as Ecosystem items (Monai, Nussl, Code-Gen).

**Comment by trsvchn** [comment]
Thanks @ydcjeff ! Looks cool!

**Pull request merged**